linter_base/src/lints.scala

```diff
@@ -596,6 +806,23 @@ object Complex_Method extends AST_Lint
   val name: String = "complex_method"
   val severity: Severity.Level = Severity.Warn
 
+  val short_description: Lint_Description =
+    Lint_Description.empty
+      .add("Using complex methods (e.g. too much nesting) should be avoided.")
+
+  val long_description: Lint_Description =
+    Lint_Description.empty
+      .add("This lint warns users from using complex method. A method is considered complex,")
```

Comment:
More concise: "Warns users from using overly complex methods, i.e. if either of the following holds: "